app/controllers/v0/facilities/ccp_controller.rb

```diff
@@ -43,9 +43,13 @@ def search_params
     params.permit(:type, :address, services: [], bbox: [])
   end
 
+  def trim_params
+    params.permit(:trim)
```

Comment:
it looks like everywhere this is used, `[:trim]` is accessed.  Maybe this method should do that?

Reply:
moved this to a feature flag